doc/ref_cert/README.md

```diff
@@ -1,9 +1,136 @@
 [<< Back](../)
 
-# CNTT Reference Conformance
+# CNTT Reference Conformance (RC) Test Case Integration
 
-<a name="available-cr"></a>
+All CNTT conformance suites must fulfill the following test case integration
+requirements to guarantee a smooth overall integration, the same end user
+actions and a unique test result format (e.g. OPNFV test result database)
+needed by the end users and the test case result verification programs (e.g.
+[OVP](https://www.opnfv.org/verification)). Historically, these rules were
+agreed by RC1 team and have been applied since
+[CNTT Snezka](https://github.com/cntt-n/CNTT/wiki/Snezka). The new RC2 stream
+asks to move them in this common place applicable to all CNTT conformance
+suites.
+
+<a name="ri-rc-toolchaings"></a>
+## CNTT RI and RC toolchains
+
+[OPNFV](https://www.opnfv.org/) has built a complete CI/CD toolchain
+for continuously deploying and testing cloud infrastructure.
+
+As for all OPNFV installer projects,
+[Jenkins](https://build.opnfv.org/ci/view/cntt/) triggers scenario deployments,
+runs the OPNFV gating test cases and then publishes all test results in the
+[centralized test database](https://docs.opnfv.org/en/stable-hunter/_images/OPNFV_testing_working_group.png)
+and all artifacts (reports, logs, etc.) to
+[an S3 compatible storage service](http://artifacts.opnfv.org/).
+
+The CNTT verification, validation and conformance processes will leverage
+existing OPNFV testing knowledge (projects) and experience (history) and then
+will conform to the overall toolchain design already in-place. The RC toolchain
+only requires for the local deployment of the components instead of leveraging
+the common OPNFV centralized services. But the interfaces remain unchanged
+mainly leveraging test jobs, the common test case execution, the test
+result database and the S3 protocol to publish the artifacts. It's worth
+mentioning that dumping all results and logs required for conformance is
```

Comment:
```suggestion
existing OPNFV testing knowledge (projects) and experience (history) and conform to the overall toolchain design already in-place. The RC toolchain
only requires the local deployment of each component rather than leveraging
the common OPNFV centralized services. However, the interfaces remain unchanged
mainly leveraging test jobs, the common test case execution, the test
result database, and the S3 protocol to publish the artifacts. It's worth
mentioning that dumping all results and logs required for conformance is
```

Reply:
@xmulligan Could you please confirm it's just about removing 'will'? your proposal is also breaking the format.

Reply:
correct, only removing 'will' didn't mean to break the formatting

Reply:
done.